app/scripts/modules/core/src/application/config/ApplicationConfig.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { Overridable, IOverridableProps } from 'core/overrideRegistry';
+import { AngularJSAdapter } from 'core/reactShims';
+
+export interface IApplicationConfigDetailsProps extends IOverridableProps {
+}
+
+@Overridable('applicationConfigView')
+export class ApplicationConfig extends React.Component<IApplicationConfigDetailsProps> {
+  public render() {
+    const templateUrl = require('./applicationConfig.view.html');
+    return (
+      <AngularJSAdapter {...this.props} templateUrl={templateUrl} controller="ApplicationConfigController" controllerAs="config" />
```

Comment:
Provide a default components for when no override is configured.
The default component renders the AngularJS template/controller